### PR TITLE
Feat admin analytics

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: app/imports/startup/server/publications.js [hunk 1]
```diff
@@ -72,11 +72,27 @@ Meteor.publishComposite('admin.application.single', function(applicationId) {
...
+});
+
+Meteor.publishComposite('admin.applications.latest', function() {
   return {
 
     find() {
-      return Applications.find({});
+      return Applications.find({}, {
+        fields: {
+          userId: 1,
```
Comment (varffs) [inline app/imports/startup/server/publications.js:89]: doesnt this need the createdAt published too so client sort can be correct?

File: app/imports/components/admin/adminIndexAnalytics.jsx [hunk 2]
```diff
@@ -1,17 +1,84 @@
...
-    let totalApplications = this.props.applications.length;
+    let total = this.props.total;
+    let stats = this.props.stats;
+    let roundedStats = this.roundStats(stats, total, 0.12);
 
     return (
-      <div className='row margin-bottom-mid'>
-        <section className='fluid-col s-12 m-12'>
+      <section className='row margin-bottom-mid'>
+        <div className='fluid-col s-12 m-12'>
```
Comment (varffs) [inline app/imports/components/admin/adminIndexAnalytics.jsx]: doesnt need `m-12` if `s` is 12 too

File: app/imports/components/admin/adminIndexAnalytics.jsx [hunk 3]
```diff
@@ -1,17 +1,84 @@
 import React, { Component } from 'react';
 
 export class AdminIndexAnalytics extends Component {
+  getPercentage(part, total) {
```
Comment (varffs) [inline app/imports/components/admin/adminIndexAnalytics.jsx]: not a great name is it more like `renderPercentage` or something

File: app/imports/components/admin/adminIndexLayout.jsx [hunk 4]
```diff
@@ -28,6 +28,56 @@ export class AdminIndexLayout extends Component {
     i18n.offChangeLocale(this.localeChanged.bind(this));
   }
 
+  /** 
+   * Return the count of application with specified status.
```
Comment (varffs) [inline app/imports/components/admin/adminIndexLayout.jsx]: if you gonna document functions like this (pretty good idea I think) you should do it for the `roundStats` one above too

**Comment by varffs** [comment]
The code reads great other than a few bits I commented on. I don't have enough applications in my local to test it well (we could do with a dummy set for applaya in general maybe?) but your testing was working well. The styling isn't perfect esp on mobile but I think that could be another feature branch as it would be good to get this live and see it in practice to influence the design imo

File: app/imports/startup/server/publications.js (hunk 1 above)
Comment (mixcas) [reply app/imports/startup/server/publications.js:89]: maybe, ill check

**Comment by mixcas** [comment]
@varffs u can download data from s3 bucket and run the mongo-restore script

**Comment by mixcas** [comment]
@varffs rdy

**Pull request merged**